### PR TITLE
Add edit command feature

### PR DESCRIPTION
**Pull request by daredevil999**
In Progress

### CODE REVIEW

**Comment by daredevil999** [comment]
Ready for review

File: src/seedu/addressbook/commands/EditCommand.java [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    public static final String MESSAGE_USAGE = COMMAND_WORD 
+            + ":\n" + "Edits a person in the address book identified by the index number in the last person listing. "
+            + "Can edit contact details but not tags. Edited contacts are not private\n\t"
+            + "Parameters: PERSON INDEX p/PHONE e/EMAIL a/ADDRESS \n\t"
+            + "Example: " + COMMAND_WORD
+            + " 1 p/98765432 e/johnd@gmail.com ";
+
+    public static final String MESSAGE_SUCCESS = "Person edited: %1$s";    
+    
+    private Phone newPhone = new Phone(DEFAULT_PHONE_STRING, false);
```
Comment (rickyyx) [inline src/seedu/addressbook/commands/EditCommand.java]: just curious, will initialise all these unedited fields in constructor to null work?

**Review by rickyyx — COMMENTED** [review]
Looks good to me

File: src/seedu/addressbook/commands/EditCommand.java (hunk 1 above)
Comment (daredevil999) [reply src/seedu/addressbook/commands/EditCommand.java]: hey, yeah it works! The unedited fields are now initialised to be null in constructor.

**Pull request merged**